modules/howtos/examples/caching_async.py

```diff
@@ -0,0 +1,83 @@
+import asyncio, time, aiofiles
+from sanic import Sanic, response
+from acouchbase.bucket import Bucket
+from couchbase.exceptions import *
+from couchbase.options import Durations
+
+EXPIRY = Durations.minutes(1)
```

Comment:
Bear in mind this is likely to change to use time_delta to be more Python idiomatic, but it should be a simple change to update.